erpnext/setup/doctype/employee/employee.json

```diff
@@ -212,7 +212,7 @@
    "label": "Status",
    "oldfieldname": "status",
    "oldfieldtype": "Select",
-   "options": "Active\nInactive\nSuspended\nLeft",
+   "options": "Active\nInactive\nSuspended\nEnded",
```

Comment:
I don't think "Ended" is the correct word for Employee. We say employee is inactive, suspected or left
Not employee ended 🫠

Reply:
I understand your point
Left is impossible to translate in other languages than English
In this case it has two different meanings : Left as direction in Frappe and status in ERPNext and HRMS
and on top is "options" in Frappe and ERPNext    
@barredterra maybe we change Left to Left Align in Frappe ?

Reply:
@mahsem "Left align" sounds good to me  👍

Reply:
@barredterra  #28871

Reply:
@barredterra 
https://github.com/frappe/frappe/pull/28871

Reply:
@barredterra https://github.com/frappe/frappe/pull/28871